src/zkevm_specs/evm/opcode.py

```diff
@@ -377,6 +377,13 @@ def stack_underflow_pairs() -> List[Tuple[Opcode, int]]:
     return pairs
 
 
+def stack_pairs() -> List[Tuple[Opcode, int, int]]:
```

Comment:
This function returns a triplet instead of a pair, so maybe a different name would be more appropriate?

Reply:
renamed to `stack_bounds` .